marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/entities/entity-search-lib.sjs

```diff
@@ -342,17 +345,23 @@ function addEntitySpecificProperties(result, entityInfo, selectedPropertyMetadat
         result.entityProperties.push(getPropertyValues(parentProperty, entityInstance));
       });
 
+      propertyMetadata.forEach(parentProperty => {
+        result.entityInstance.push(getPropertyValues(parentProperty, entityInstance));
```

Comment:
I thought the intent was that the entire entity instance would be included - i.e. the part at /*:envelope/*:instance ? Isn't that what the UI needs for the current row expander, where it's just showing a pretty JSON view of the entity? I know in the future, we want to show properties with labels for the row expander and detail view, but I think that's 5.4.0.

Reply:
Yes, propertyMetadata has all the entityProperties of an entityModel.
Row expander shows a tabular column with entityTypePropertyName and its corresponding value.

Reply:
Doesn't that result in a change of what's shown though? The row expander is showing a pretty JSON view of the entity instance. That's apparent because the user sees "shipping / Address / street" instead of just "shipping / street". 

I'll try this out locally, it just seems the easiest thing to do is return the entity instance so that the UI doesn't have to try to guess where it is in the extracted content array.